docs/source/modules.rst

```diff
@@ -0,0 +1,146 @@
+Modules
+=======
+
+While different modules perform different jobs, their interfaces all follow
+the same pattern as much as possible.
+
+The API of each module is composed of two parts. The *auth* parameter contains
+the pieces of information that are related to the Sensu Go backend that the
+module is connecting to. All other parameters hold the information related to
+the resource that we are operating on. Typical task would look like this:
+
+.. code-block:: yaml
+
+   - name: Make sure asset is present
+     asset:
+       auth:
+         url: http://sensu.host.com:8080
+         user: my-user
+         password: pass
+       namespace: my-namespace
+       name: my-asset-name
+       builds: ...
+
+Almost every module for manipulating resources has its info counterpart that
+can be used to retrieve information about the certain resources. For example,
+to get a list of assets from the ``my-namespace`` namespace, we would write a
+task like this one:
+
+.. code-block:: yaml
+
+   - name: Fetch list of all assets
+     asset:
+       auth:
+         url: http://sensu.host.com:8080
+         user: my-user
+         password: pass
+       namespace: my-namespace
+     register: result
+
```

Comment:
Here, I would add a note "Note the usage of the _asset_info_ module in the example above".